pootle/apps/pootle_app/management/commands/update_stores.py

```diff
@@ -51,10 +54,38 @@ def handle_translation_project(self, translation_project, **options):
         return False
 
     def handle_store(self, store, **options):
-        store.update(
-            overwrite=options['overwrite'],
-            only_newer=not options['force']
+        if not store.file:
+            return
+        disk_mtime = store.get_file_mtime()
+        if not options["force"] and disk_mtime == store.file_mtime:
+            # The file on disk wasn't changed since the last sync
+            logger.debug(u"File didn't change since last sync, skipping "
+                         u"%s" % store.pootle_path)
+            return
+        if options["overwrite"]:
+            store_revision = store.get_max_unit_revision()
+        else:
+            store_revision = store.last_sync_revision or 0
+
+        changes = store.update(
+            store.file.store,
+            store_revision=store_revision,
         )
+        store.file_mtime = disk_mtime
+        if changes and any(x > 0 for x in changes.values()):
+            update_unsynced = None
+            if store.last_sync_revision is not None:
+                updated_unsynced = (
+                    store.increment_unsynced_unit_revision(
+                        store.get_max_unit_revision()))
```

Comment:
When calling functions/methods by breaking lines, let's keep the final `)` in its own line. The same applies everywhere in this PR.